packages/app-store/fathom/components/EventTypeAppCardInterface.tsx

```diff
@@ -4,19 +4,17 @@ import useIsAppEnabled from "@calcom/app-store/_utils/useIsAppEnabled";
 import type { EventTypeAppCardComponent } from "@calcom/app-store/types";
 import { TextField } from "@calcom/ui";
 
-import type { appDataSchema } from "../zod";
+import { appDataSchema } from "../zod";
 
 const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({ app, eventType }) {
-  const [getAppData, setAppData, LockedIcon, disabled] = useAppContextWithSchema<typeof appDataSchema>();
+  const { getAppData, setAppData, disabled } = useAppContextWithSchema<typeof appDataSchema>();
```

Comment:
This is an example where we don't need the LockedIcon so with an object we can call on what we do need. Also makes it easier if we have to add more data to the context in the future.

https://github.com/calcom/cal.com/pull/9950#discussion_r1253492838